### ISSUE TITLE
no db found

### ISSUE THREAD

**Issue by binarygeek119** [Cloud9Developer/Jellyfin-Newsletter-Plugin#65]
im on Jellyfin Server with Media Segments API docker by [endrl](https://github.com/endrl)
v.  10.9.0

```
[06:03:29] [ERR] [117] Jellyfin.Api.Middleware.ExceptionMiddleware: Error processing request: Could not load file or assembly 'SQLitePCL.pretty, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
. URL POST /Smtp/SendTestMail.

```

**Comment by Cloud9Developer** [comment]
I've not done any testing on the beta release of jellyfin (10.9.0), so I assume the issue lies there somehow since from what I can tell, test mail still works on 10.8.12

**Comment by Cloud9Developer** [comment]
Or, they may be swapping DB providers in the next major release, so that could be the issue too. In which case, I'll update when the time comes to use the new database provider. But for now, if that is the actual cause, there is no immediate plan to fix for the beta version of Jellyfin

**Comment by binarygeek119** [comment]
It's getting closer to being released. It's now in the beta unstable branch. And they are shooting for end of April for stable release.

**Comment by Cloud9Developer** [comment]
This was, in fact, due to the updates to Jellyfin backend. Now that it's been officially release, I've been forced to fix it ;) so the latest version 0.6.2.0 should work for you now!
